### DISCUSSION TITLE
DMO/ VDO from 1 July

### DISCUSSION THREAD

**Discussion by purcell-lab** [amberelectric/public-api#117] [Q&A] [answered]
Myles,

With the updated default network charges from 1 July has the API been updated to reflect these changes?

In particular in the past very_low (green) used to be below DMO and low (yellow) used to be above DMO. Have the colours in the app been updated to reflect the new DMO/ VDO?

I know in my case the DMO now also depends on annual consumption, so will this be reflected in the API?

Would it also be possible to expose the current DMO/ VDO for a site via the API?

**Comment by oldwindmill** [comment]
to my mind, any price above dmo ought to be amber at least.

**Comment by purcell-lab** [comment, marked as answer]
To answer some of my own question looks like the Price Descriptor 
![image](https://user-images.githubusercontent.com/79175134/178405819-58e6b3c4-4031-47e5-b44a-0242ed9c3541.png)

and app colours are in alignment and represent the new DMO:

![image](https://user-images.githubusercontent.com/79175134/178405957-e6fe7848-2048-46ce-997a-a847ccad996c.png)

1030-1200 - very_low = Green
1230-1400 - high = Red
1000-1030 - low = Yellow
1200-1230 - neutral = Orange

**Comment by oldwindmill** [comment]
i've had a look at the current app colours just now. i see '40c' is labelled 'low' even though that is double the vdo for my distributor. even if it was peak, it would only just match the vdo - hardly 'low'. although compared to the rest of the day, currently sitting at $1.10 i guess it is. 
i hope those generators get their act together very soon. i dont understand how retailers can stay in business buying at that rate and selling at 0.20....
